src/message.rs

```diff
@@ -201,17 +219,20 @@ pub enum Content {
 impl MessageKind {
     /// New message
     pub fn new(
-        request_id: Option<u64>,
         id: Option<Uuid>,
+        request_id: Option<u64>,
+        trade_index: Option<u32>,
```

Comment:
If the trade index was signed, a -1 could be used to indicate that the user want in to the anonymous mode and no reputation will be tracked. Or is this being dealt in any other way?

Reply:
for anonymous operation we just send the message without `trade_index` or `trade_index: null`

I added an example here:

https://mostro.network/protocol/key_management.html#full-privacy-mode